About/whitepaper.md

```diff
@@ -22,23 +22,23 @@ sub: '
   </a>
 </div>
 
-seL4 is a high-assurance, high-performance operating system microkernel. It is
-unique because of its comprehensive formal [verification](../Verification/),
-without compromising [performance](Performance/). It is meant to be used as a
+seL4 is a high-assurance, high-performance operating system microkernel. Its
+uniqueness lies in the comprehensive formal [verification](../Verification/),
+while maintaining high [performance](Performance/). seL4 was designed as a
 trustworthy foundation for building safety- and security-critical systems. It is
 available as open source on [GitHub](https://github.com/seL4/) and supported by
 the [seL4 Foundation](../Foundation/).
 
-Being a *kernel* means it is the piece of software that runs at the
-heart of any software system and controls all accesses to resources. It
-provides fine-grained access control through
-[capabilities](https://en.wikipedia.org/wiki/Capability-based_security),
-and controls communication between components of the system. It is the
-most critical part of the software system, and runs in privileged mode.
+As a *kernel*, it is the piece of software that runs at the heart of any software
```

Comment:
This destroys the parallel beginning of the next paragraph. I'm not really happy with either version, though, tbh. I'm Ok with `Being a *kernel*` and `Being a *microkernel*`, but `runs at the heart of any software system` is of course not true. If it runs in a system, it runs at the heart of that system, but it doesn't run in any system. Needs more thinking.

Reply:
We could recreate the parallel beginning with `seL4 is a *kernel*, ..` and `seL4 is a *microkernel, ...`. Could even be a new sentence instead of a `, which means`. E.g.

```md
seL4 is a *kernel*. This means it is at the heart of any software system it runs in,
and controls all access to  resources. It provides fine-grained control for that
access through ...

seL4 is also a *microkernel*. This means that it is reduced to a minimal
policy-free core and can therefore form a dependable basis for building
arbitrary systems in many different usage scenarios.
```

Reply:
I like Gerwin's latest version.